Menu.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PELICULAS_MENU_H
+#define PELICULAS_MENU_H
+
+#include "Lista.h"
+#include <fstream>
+#include <sstream>
+
+using namespace std;
+
+class Menu{
+
+private:
+
+Lista<string> ListaPV;
+
+Lista<string> ListaPNV;
+
+Lista<string> ListaPR;
```

Comment:
```suggestion
Lista<Pelicula> peliculasVistas;
Lista<Pelicula> peliculasNoVistas;
Lista<Pelicula> peliculasRecomendadas;
```
Idem anterior. La lista es de películas, y el cambio en el nombre de las listas no es necesario.